src/transactions/programs/MemoProgram.ts

```diff
@@ -0,0 +1,12 @@
+import { PublicKey } from '@solana/web3.js';
+import { Program } from '../../accounts';
+
+export class MemoProgram extends Program<{}> {
+  static readonly PUBKEY = new PublicKey('MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr');
+
+  constructor() {
+    super(MemoProgram.PUBKEY);
+  }
+}
```

Comment:
I didn't see the memo program anywhere (the rest, like TOKEN, had PubKeys in the web3 lib)